playwright-gh-pages/deploy-report-pages/build-markdown-table.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
```

Comment:
this script is getting rather complex to follow and maintain in bash. I advice you to re-write it into js, or go and then run it from your action (after setup/node or setup/go).

this will make it much easier for anyone to maintain it in the future that is not so comfortable with bash

Reply:
Yeah totally agree! Addressed in https://github.com/grafana/plugin-actions/pull/50/commits/5121a9697a73e0630ec617756c5eb4155864be54